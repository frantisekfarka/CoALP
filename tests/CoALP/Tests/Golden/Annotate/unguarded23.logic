q(X) :- q(a).
q(a).
