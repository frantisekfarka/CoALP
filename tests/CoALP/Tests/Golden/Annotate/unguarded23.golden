q(V_2, transform-func-1(V_-3)) :- q(a, V_-3).
q(a, transform-func-2).
